composer.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "sectionio/metrics",
-  "description": "Improve site metrics for your Section.io Magento environment.",
+  "description": "Improve website performance metrics for the Section.io Magento environment.",
   "type": "magento2-module",
-  "version": "103.0.5",
+  "version": "104.0.5",
   "license": [
     "OSL-3.0",
     "AFL-3.0"
   ],
   "require": {
-    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0|~7.3.0",
-    "magento/framework": "~100.0 || ~101.0 || ~102.0|| ~103.0"
+    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0",
+    "magento/framework": "~100.0 || ~101.0 || ~102.0|| ~103.0|| ~104.0"
```

Comment:
Due to the above varnish constant I think we need to declare that we require `magento/framework` v102.0.3 or later and `magento/module-page-cache` v100.3.3 or later.

https://packagist.org/packages/magento/community-edition#2.3.3

Reply:
I pushed this require block instead

```
  "require": {
    "php": "~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0",
    "magento/framework": "~102.0.3|| ~103.0|| ~104.0",
    "magento/module-page-cache": "~100.3.3"
  },
```